src/main/java/webserver/http/response/HttpResponse.java

```diff
@@ -0,0 +1,166 @@
+package webserver.http.response;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import utils.FileIoUtils;
+import webserver.http.ContentType;
+import webserver.http.Cookie;
+import webserver.http.HttpStatus;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URISyntaxException;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Objects;
+
+import static model.Constant.*;
+
+public class HttpResponse {
+    private static final Logger logger = LoggerFactory.getLogger(HttpResponse.class);
+
+    private static final String RESOURCES_TEMPLATES = "./templates";
+    private static final String RESOURCES_STATIC = "./static";
+    public final static String EXTENSION_SPERATOR = ".";
+    public static final String CONTENT_LENGTH = "Content-Length";
+    public static final String HEADER_KEY_VALUE_SEPARATOR = ": ";
+    public static final String LINE_SEPARATOR = "\r\n";
+    public static final String HEADER_SEPARATOR = " ";
+
+
+    private DataOutputStream out;
+    private ResponseLine responseLine;
+    private ResponseHeader responseHeader;
+    private ResponseBody responseBody;
+    private Cookie cookie;
+
+    public HttpResponse(ResponseLine responseLine, ResponseHeader responseHeader) {
+        this.responseLine = responseLine;
+        this.responseHeader = responseHeader;
+        this.responseBody = new ResponseBody(new byte[0]);
+        this.cookie = new Cookie(Collections.emptyMap());
+    }
+
+    public HttpResponse(ResponseLine responseLine, ResponseHeader responseHeader, Cookie cookie) {
+        this.responseLine = responseLine;
+        this.responseHeader = responseHeader;
+        this.responseBody = new ResponseBody(new byte[0]);
+        this.cookie = cookie;
+    }
```

Comment:
```suggestion
    public HttpResponse(ResponseLine responseLine, ResponseHeader responseHeader, Cookie cookie) {
        this(responseLine, responseHeader, new ResponseBody(new byte[0]), cookie);
    }
```

```suggestion
    public HttpResponse(ResponseLine responseLine, ResponseHeader responseHeader) {
        this(responseLine, responseHeader, new ResponseBody(new byte[0]), new Cookie(Collections.emptyMap()));
    }
```
다음 생성자 추가한 후에 위와 같이 구현

public HttpResponse(ResponseLine responseLine, ResponseHeader responseHeader, ResponseBody responseBody, Cookie cookie) {
}